cpp/include/cudf/detail/search.hpp

```diff
@@ -81,4 +81,21 @@ bool contains_nested_element(column_view const& haystack,
                              column_view const& needle,
                              rmm::cuda_stream_view stream);
 
+/**
+ * @brief Check if each row of the `needles` column exists in the `haystack` column.
+ *
+ * This function is designed for nested types only. It can also work with non-nested types
+ * but with lower performance due to the complexity of the implementation.
+ *
+ * @param haystack The column containing search space.
```

Comment:
Following the documentation guidelines below, periods are not needed for `@throw`, `@tparam`, `@param`, or `@return`. The general rule is that complete sentences need periods, but sentence fragments (like the content of a bulleted list or these parameter descriptions) do not need a period.

https://github.com/rapidsai/cudf/blob/branch-22.08/cpp/docs/DOCUMENTATION.md